pkg/oam/env.go

```diff
@@ -140,7 +141,17 @@ func DeleteEnv(envName string) (string, error) {
 	if err != nil {
 		return message, err
 	}
-	if err = os.RemoveAll(filepath.Join(envdir, envName)); err != nil {
+	envPath := filepath.Join(envdir, envName)
+	if _, err := os.Stat(envPath); err != nil {
+		if os.IsNotExist(err) {
+			envExist = false
+			message = envName + " does not exist"
```

Comment:
you could directly return this error

Reply:
if return this error directly,command line will display `Error: stat /Users/admin/.vela/envs/tessdf: no such file or directory`,it's ok?

Reply:
return  envName + " does not exist"

Reply:
Did it